### ISSUE TITLE
About the Mean and Std during Fine-tuning SOLIDER

### ISSUE THREAD

**Issue by AyanamiReiFan** [VlSomers/keypoint_promptable_reidentification#12]
The input normalization mean and std are [0.5, 0.5, 0.5], which is the same as in the SOLIDER-REID repository. However, after checking the SOLIDER repository, I found that during pretraining, the normalization used was the commonly seen ImageNet norm: [0.485, 0.456, 0.406] for the mean and [0.229, 0.224, 0.225] for the std . I also checked other downstream task repositories under SOLIDER, and they also seem to use the ImageNet norm. 
I asked this question in the SOLIDER-REID repository, but I haven't received a response yet. 

You mentioned that SOLIDER is difficult to fine-tune—could this be related? Have you tried using the ImageNet norm?

**Comment by VlSomers** [comment]
Hi @AyanamiReiFan, I think I tried the ImageNet norm at some point (by error) and got slightly worse results. However, I'm not sure at all about this, this is just what I have in my memory from all the experiments I did last year! I don't think it should make a huge difference. When I said SOLIDER was hard to fine-tune, I was mainly referring to adding both a prompt tokenizer upstream to the backbone, and a new part-based head downstream, both not being pre-trained compared to the backbone. It was hard to train this new pipeline without having some sort of catastrophic forgetting in the backbone, leading to a huge performance drop.
